bioframe/ops.py

```diff
@@ -511,8 +510,8 @@ def overlap(
     index_col = return_index if isinstance(return_index, str) else "index"
     index_col_1 = index_col + suffixes[0]
     index_col_2 = index_col + suffixes[1]
-    df_index_1 = pd.DataFrame({index_col_1: df1.index[events1]})
-    df_index_2 = pd.DataFrame({index_col_2: df2.index[events2]})
+    df_index_1 = pd.DataFrame({index_col_1: df1.index[events1]}, dtype=pd.Int64Dtype())
```

Comment:
I believe @nvictus introduced the `_to_nullable_dtype()` in the last update... maybe there is a way we can remove boilerplate & also use same strategy for closest ?

Reply:
We can, but I don't see how it will help in this case, frankly. It's storage of the index that is always integer and not absent.

Reply:
Yup, I agree that we do not seem to need a nullable type here.